### PR TITLE
Feature add skilavottord to codeowners

### PR DESCRIPTION
**Pull request by qdd1**
# \<Description\>

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -51,3 +51,5 @@
 
 /libs/application/templates/document-provider-onboarding/ @island-is/advania
 /apps/services/xroad-collector                            @island-is/advania
+
+/apps/skilavottord/                                       @island-is/deloitte
```
Comment (darrikonn) [inline .github/CODEOWNERS:55]: missing /libs/skilavottord/

**Review by pshomov — COMMENTED** [review]
Also /libs/skilavottord/

File: .github/CODEOWNERS (hunk 1 above)
Comment (qdd1) [reply .github/CODEOWNERS:55]: Added

**Review by pshomov — APPROVED** [review]
LGTM

**Review by darrikonn — APPROVED** [review]

**Pull request merged**